src/components/layout/Header.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Image, Menu } from 'semantic-ui-react';
+
+class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 'home',
+    };
+
+    this.handleItemClick = this.handleItemClick.bind(this);
+  }
+
+  handleItemClick(e, { name }) {
+    this.setState({ activeItem: name });
+  }
+
+  render() {
+    const { activeItem } = this.state;
+    return (
+      <Menu pointing secondary>
+        <Menu.Item
+          as={Link}
+          to="/"
+          name="home"
+          active={activeItem === 'home'}
+          onClick={this.handleItemClick}
+        >
+          <Image
+            size="mini"
+            bordered
+            centered
+            circular
+            src="/images/logo.png"
```

Comment:
@koitoror you will have to import the image from the images folder and pass it to the Image component as a component `src={logo}`

Reply:
Done